### DISCUSSION TITLE
Maintenance status

### DISCUSSION THREAD

**Discussion by andrewdavidmackenzie** [plotters-rs/plotters#627] [General]
What's the maintenance status of plotters?

We're updating our app that uses it to the 0.13 version of Iced and running into issues, and we wondered if it's actively maintained and if tracking changes in Iced is something we should expect or can rely on?

Thanks for any comments

**Comment by AaronErhardt** [comment]
Yes, plotters is maintained. I merge PRs with improvements and fixes and do releases from time to time. There's no official integration for iced though, so I can't comment on that. There only seems to be a third-party crate that adds a iced backend to plotters.

**Comment by andrewdavidmackenzie** [comment]
Thanks. I asked in that repo also!
